strawberry_django/type.py

```diff
@@ -178,7 +184,10 @@ def process_type(
     if not hasattr(cls, "is_type_of"):
         cls.is_type_of = lambda obj, _info: isinstance(obj, (cls, model))
 
-    strawberry.type(cls, **kwargs)
+    # Get type description from either kwargs, or the model's docstring
+    description = kwargs.pop("description", cleandoc(model.__doc__))
```

Comment:
Just to make sure we don't add an empty string here

```suggestion
    description = kwargs.pop("description", cleandoc(model.__doc__) if model.__doc__ else None)
```

Reply:
Here's my implementation: https://github.com/strawberry-graphql/strawberry-graphql-django/pull/160/commits/d52cfc7200aa7ed911f416299aa0b3a0f2535c89